apps/renterd/contexts/config/transformDown.ts

```diff
@@ -148,10 +149,13 @@ export function transformDownGouging({
       scDecimalPlaces
     ), // TB/month
     maxUploadPriceTB: toSiacoins(
-      new BigNumber(gouging.maxUploadPrice),
+      valuePerByteToPerTB(new BigNumber(gouging.maxUploadPrice)),
+      scDecimalPlaces
+    ),
+    maxDownloadPriceTB: toSiacoins(
```

Comment:
I think the logic is correct but I was expecting these `XPriceTB` fields to get renamed as well?

Reply:
These fields are names based on the units used client-side, which havent changed since I adjusted the conversions. eg: maxUploadPrice used to be per TB on the server, now its per byte - the client was per TB and with this change is kept per TB.